source/topic4.rst

```diff
@@ -199,7 +199,7 @@ Nested conditionals
         else:
             print('Third Quadrant')
 			
-* In the above example, we *could* have done it without nesting and using just ``and`` and whatnot, but the nesting does make things easy to follow. 
+* In the above example, we *could* have done it without nesting by using ``and`` and whatnot, but for fun we can nest it
 
 * Again, no limit to how deep you nest... but mind the readability of your code!
```

Comment:
Python does actually have a limit to the amount of nesting, it's 20. [StackOverflow question](https://stackoverflow.com/questions/44972719/why-does-python-have-a-limit-on-the-number-of-static-blocks-that-can-be-nested) has some context.

Reply:
Oh shit, I totally remember learning this earlier this year.

Reply:
Really? Good to know.

Reply:
Neat, eh!?